client/src/components/_common/SectionContent/SectionContent.layouts.unequalColumns.md

```diff
@@ -0,0 +1,90 @@
+# `<SectionContent>` Layouts with Unequal Columns
```

Comment:
__Explanation__
Support different widths via CSS multi-column is not re-usable without some assembly. So, I wrote instructions.

<details><summary>Details</summary>

Two sections, `<Dashboard>` and `<ManageAccount>`, have challenging layouts—column border __and__ different-width columns _and_ item re-order depending on screen width. Supporting them in a re-usable manner was not 100% achieved. Some assembly is required __only__ for the layouts' different widths.

</details>